paper.md

```diff
@@ -27,10 +27,12 @@ authors:
     affiliation: 10
   - name: Fada Guan
     affiliation: 11
+  - name: Nicola Lambri
+    affiliation: "12, 13"
   - name: Matthew Sobolewski
-    affiliation: "8, 12"
+    affiliation: "8, 14"
   - name: Cody Crewson
```

Comment:
@crcrewso, would you be able to bump yourself up one. @msobolewski there is taking the final spot on the author list as an overarching project lead from the very early days (before it was even called PyMedPhys :slightly_smiling_face:)

Reply:
Absolutely, that's a big faux pas on my part. Sorry @msobolewski for missing that one.

Reply:
All good :) easy fix :)